components/ntp_background_images/browser/ntp_background_images_source.cc

```diff
@@ -70,9 +70,9 @@ void NTPBackgroundImagesSource::StartDataRequest(
   auto* images_data =
       service_->GetBackgroundImagesData(IsSuperReferralPath(path));
   if (!images_data) {
-    base::PostTask(FROM_HERE,
-        base::BindOnce(std::move(callback),
-                       scoped_refptr<base::RefCountedMemory>()));
+    base::PostTask(FROM_HERE, {base::ThreadPool()},
```

Comment:
I believe this should also be posting to the UI thread

Reply:
Addressed by 3e9dede3537553d72fa9d29cc9c02b8c11a8757f